x/staking/keeper/params.go

```diff
@@ -40,11 +40,8 @@ func (k Keeper) BondDenom(ctx sdk.Context) (res string) {
 }
 
 // PowerReduction - is the amount of staking tokens required for 1 unit of consensus-engine power.
-// Currently, this returns a global variable that the app developer can tweak.
-// TODO: we might turn this into an on-chain param:
-// https://github.com/cosmos/cosmos-sdk/issues/8365
 func (k Keeper) PowerReduction(ctx sdk.Context) sdk.Int {
```

Comment:
```suggestion
func (k Keeper) PowerReduction() sdk.Int {
```

Reply:
I think its best to keep this to cause the least amount of breakage in The API. I know we don't need it now but we may need in the future if its dynamic. Would rather avoid to breaks

Reply:
ok, added an underscore